website/docs/installation/documentdb/deb.md

```diff
@@ -4,6 +4,15 @@ sidebar_position: 2
 
 # DEB package
 
+:::note
+We provide different `.deb` packages for various deployments.
+
+- The development package (contains either `-dev` or `-dbgsym` suffix e.g., `documentdb-dev.deb`/`documentdb-dbgsym.deb`) is for debugging purposes.
+  It includes features that significantly slow performance and is not recommended for production use.
+- For other use cases, we recommend the production package (e.g., `documentdb.deb`).
```

Comment:
I think production explanation should come before development. Because we want to steer away from using development package.